include/nsysnet/nssl.h

```diff
@@ -4,46 +4,318 @@
 /**
  * \defgroup nsysnet_nssl Nintendo SSL
  * \ingroup nsysnet
+ *
+ * The NSSL library is Nintendo's interface for making SSL connections on the
+ * Wii U. To use NSSL directly, call NSSLInit() then NSSLCreateContext() to
+ * create a context. This context - after adding root CAs via NSSLAddServerPKI()
+ * or NSSLAddServerPKIExternal() - can be used to create connections
+ * (NSSLCreateConnection()) that can transfer data using NSSLRead() and
+ * NSSLWrite().
+ *
+ * For HTTPS connections, consider using libcurl.
+ *
+ * \if false
+ * meta: how to link to libcurl?
```

Comment:
It's [nlibcurl](https://github.com/decaf-emu/wut/blob/master/cafe/nlibcurl.def)

see https://github.com/decaf-emu/wut/tree/master/samples/curl